apps/web-giddh/src/app/vouchers/template/all-templates/templates/gst-template-a/gst-template-a.component.ts

```
@@ -0,0 +1,237 @@
+                    if (template.sections.footer.data.imageSignature?.display) {
+                        this.showImageSignature = true;
+                        if (template.sections.footer.data.imageSignature.label) {
+                            this.imageSignatureSrc = ApiUrl + 'company/' + this.companyUniqueName + '/image/' + template.sections.footer.data.imageSignature.label;
+                        } else {
+                            this.imageSignatureSrc = '';
+                        }
+                    } else {
+                        this.showImageSignature = false;
+                        this.imageSignatureSrc = '';
+                    }
+                } else if (template.sections && template.sections.footer.data.slogan?.display) {
```

Biome (2.1.2)

[error] This branch can never execute. Its condition is a duplicate or covered by previous conditions in the if-else-if chain.



(lint/suspicious/noDuplicateElseIf)